content/id/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Kustomisasi Service DNS
+content_type: task
+min-kubernetes-server-version: v1.12
+---
+
+<!-- overview -->
+Laman ini menjelaskan cara mengonfigurasi DNS
+{{< glossary_tooltip text="Pod" term_id="pod" >}} kamu dan menyesuaikan
+proses resolusi DNS pada klaster kamu.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+Klaster kamu harus menjalankan tambahan (_add-on_) CoreDNS terlebih dahulu.
+[Migrasi ke CoreDNS](/id/docs/tasks/administer-cluster/coredns/#migrasi-ke-coredns)
```

Comment:
we don't seem to have this page yet, change to EN link?

Reply:
Ok! I translated before our concession :)